data/examples/electrified_stations.json

```diff
@@ -8,6 +8,7 @@
         "cs_power_deps_depb" : 120,  // optional: maximum cs power for this station, default defined in config
         "grid_operator" : "default_grid_operator", // optional: set grid operator for cost calculation. Default is "default_grid_operator"
         "voltage_level": "MV", // optional: voltage_level for this station, default defined in config
+        "peak_load_window_power": 100,  // optional: power during peak load window (if time_windowds file given), default defined in config
```

Comment:
peak_load_window_power needs a grid operator as far as a I can see. This should be mentioned in the example file

Reply:
changed to use the correct default grid operator (consistent with grid connector info). No change to example electrified stations file.